test/manual/templates/template_api_tests.sh

```
@@ -0,0 +1,508 @@
+# ─── API Request Functions ───────────────────────────────────────────────────────
+
+api_put() {
+  local endpoint=$1
+  local data=$2
+  local additional_headers=$3
+  
+  echo "$data" | curl -s -X PUT \
+    -w "%{http_code}" \
+    -H "Content-Type: application/json" \
+    -H "Accept: application/json" \
+    $([[ -n "$additional_headers" ]] && echo $additional_headers) \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)